guides/_posts/2015-02-05-layout.html

```diff
@@ -205,34 +205,34 @@
 
 ## Intrinsic values
 
-Positioning elements with `top` and `left` properties doesn't require GSS to know the dimensions of that elements. But `right` and `bottom` properties need to know about `width` and `height` of the element respectively. Properly constrained system will provide those values indirectly with no regard to elements contents.
+Positioning elements with `top` and `left` properties doesn't require GSS to know the dimensions of those elements. But `right` and `bottom` properties need to know about `width` and `height` of the element respectively. Properly constrained systems will provide those values indirectly with no regard to element contents.
 
-However it's often desirable to know the computed dimensions of specific elements which GSS has no control over - be it text, images or blocks styled with CSS. Measurements provide dynamic "islands" to the layout, which can be automatically recomputed, whenever measured things are updated from the outside. It also may act as a feedback to the solver, disallowing content to shrink too much or helping to decide if things fit.
+However, it's often desirable to know the computed dimensions of specific elements which GSS has no control over - be it text, images or blocks styled with CSS. Measurements provide dynamic "islands" to the layout, which can be automatically recomputed, whenever measured things are updated from the outside. It also may act as a feedback to the solver, disallowing content to shrink too much or helping to decide if things fit.
 
-Prepending property with `intrinsic-` or `computed-` prefix makes GSS measure the values.
+Prepending a property with `intrinsic-` or `computed-` prefix makes GSS measure the values.
```

Comment:
This still reads a bit strange to me.

I think because we would "prepend **to**" something but also because it says that we're prepending a prefix.

Perhaps this is better?

> Prefixing a property with `intrinsic-` or `computed-` makes GSS measure the values. It also makes it *read-only* and it won't apply on the element.